README.md

```diff
@@ -1 +1,31 @@
-# issue-moderator-action
+# GitHub action to moderate issues
+
+This action allows the usage of some commands from the organization
+to moderate the issues of a repository.
+
+## Installation
+
+To configure the action simply add the following lines to your workflow file:
+
+```yml
+name: Moderator
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  moderate:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Moderate issues
+      uses: tachiyomiorg/issue-moderator-action
```

Comment:
Missing version. Unless that's actually optional?

Reply:
Don't think so, I forgot to add it.